tests/modules/assets/resources/test_assets.py

```diff
@@ -11,7 +11,7 @@ def test_find_asset(
     test_clone_submission_data,
 ):
     # Clone the known submission so that the asset data is in the database
-    clone_submission(
+    clone = clone_submission(
```

Comment:
I did wonder if this was an optimisation too far, did it cause problems with the randomisation?

Reply:
Yes